signal.c

```diff
@@ -520,6 +520,38 @@ rb_f_kill(int argc, const VALUE *argv)
     return INT2FIX(i-1);
 }
 
+/*
+ *  call-seq:
+ *     Process.kill(pid)    -> boolean
+ *
+ * Checks the given process to see whether it is alive.
+ */
+VALUE
+rb_f_exists(VALUE self, VALUE val_pid)
```

Comment:
This method is not global, use `_proc_` in name instead of `_f_`.
And I doubt that this function should be in `signal.c`.